src/models/observation.js

```diff
@@ -117,7 +117,9 @@ module.exports = (sequelize, DataTypes) => {
     classMethods: {
       associate: db => {
         Observation.belongsTo(db.Datastreams);
+        Observation.associations.Datastream.mandatory = true;
         Observation.belongsTo(db.FeaturesOfInterest);
+        Observation.associations.FeatureOfInterest.mandatory = true;
```

Comment:
Create an issue here and add a comment about this:

> SHALL link to a FeatureOfInterest entity. If no link specified, the service SHALL create a FeatureOfInterest entity from the content of the Location entities.

from table 24 http://docs.opengeospatial.org/is/15-078r6/15-078r6.html#60 

This might be the reason of this failure https://travis-ci.org/mozilla-sensorweb/sensorthings/jobs/180420687#L4775